python/pyspark/sql/connect/client.py

```diff
@@ -122,6 +122,8 @@ class ChannelBuilder:
     PARAM_TOKEN = "token"
     PARAM_USER_ID = "user_id"
     PARAM_USER_AGENT = "user_agent"
+    MAX_MESSAGE_LENGTH = 128 * 1024 * 1024
+    MAX_METADATA_LENGTH = 16 * 1024 * 1024
```

Comment:
This seems to be unused in this PR. Do we need to add this in this PR?

Reply:
No, this is a leftover. Removing.